Tests/Validation/CreatePostValidatorTests.cs

```diff
@@ -0,0 +1,66 @@
+﻿using FluentAssertions;
+using FluentValidation.TestHelper;
+using ProjectSetup.Contracts.V1.Requests;
+using ProjectSetup.Validation;
+
+namespace Tests.Validation
+{
+	public class CreatePostValidatorTests
+	{
+		private CreatePostValidator _createPostValidator;
+
+		public CreatePostValidatorTests()
+		{
+			_createPostValidator = new CreatePostValidator();
+		}
+
+		[Fact]
+		public void ValidateTextIsEmpty()
+		{
+			var createPostRequest = new CreatePostRequest { Text = "", CategoryId = Guid.NewGuid() };
+
+			var result = _createPostValidator.TestValidate(createPostRequest);
+
+			result.Errors.First().ErrorCode.Should().Be("NotEmptyValidator");
+		}
+
+		[Fact]
+		public void ValidateCategoryIdIsEmpty()
+		{
+			var createPostRequest = new CreatePostRequest { Text = "Some text", CategoryId = Guid.Empty };
+
+			var result = _createPostValidator.TestValidate(createPostRequest);
+
+			result.Errors.First().ErrorCode.Should().Be("NotEmptyValidator");
+		}
+
+		[Fact]
+		public void ValidateTextIsMoreThan400Characters()
+		{
+			var createPostRequest = new CreatePostRequest { Text = GetRandomString(401), CategoryId = Guid.NewGuid() };
+
+			var result = _createPostValidator.TestValidate(createPostRequest);
+
+			result.Errors.First().ErrorMessage.Should().Be("Text must be between 1 and 400 characters.");
+		}
+
+		private string GetRandomString(int length)
```

Comment:
You can create a helper class with this method instead of using this one. Maybe you will need this method for more cases.